toolsrc/src/vcpkg/vcpkgpaths.cpp

```diff
@@ -340,6 +340,8 @@ If you wish to silence this error and use classic mode, you can:
         packages =
             process_output_directory(filesystem, root, args.packages_root_dir.get(), "packages", VCPKG_LINE_INFO);
         scripts = process_input_directory(filesystem, root, args.scripts_root_dir.get(), "scripts", VCPKG_LINE_INFO);
+        builtin_ports =
+            process_output_directory(filesystem, root, args.builtin_ports_root_dir.get(), "ports", VCPKG_LINE_INFO);
```

Comment:
```suggestion
            process_input_directory(filesystem, root, args.builtin_ports_root_dir.get(), "ports", VCPKG_LINE_INFO);
```

this should be an input directory, no?

Reply:
It's complicated, because it's an output directory for `create`.